By Matt Jervis, 09/01/2010
updated by Karolina Lindqvist, 2011-01-31

For some reason the program prefix (ie the bit that comes before the tool name)
had been hardcoded to pic30-.  This patch returns it to what it should be
(e.g. so if compiling coff it will use the prefix pic30-coff-, etc.)


--- ./gcc/config.gcc.orig	2011-01-31 06:50:02.000000000 +0100
+++ ./gcc/config.gcc	2011-01-31 07:05:27.000000000 +0100
@@ -1562,7 +1562,7 @@
 	;;
 pic30-*-coff)
         host_can_use_collect2=no
-        host_xm_defines="PROGRAM_PREFIX=\\\"pic30-\\\" ${host_xm_defines}"
+        host_xm_defines="PROGRAM_PREFIX=\\\"${program_prefix}\\\" ${host_xm_defines}"
         host_xm_defines="OMF=\\\"coff\\\" ${host_xm_defines}"
         extra_modes=pic30/pic30-modes.h
         LANGUAGES="c"
@@ -1572,7 +1572,7 @@
         host_can_use_collect2=no
         elf=yes
         tm_defines="OBJECT_FORMAT_ELF $tm_defines"
-        host_xm_defines="PROGRAM_PREFIX=\\\"pic30-\\\" ${host_xm_defines}"
+        host_xm_defines="PROGRAM_PREFIX=\\\"${program_prefix}\\\" ${host_xm_defines}"
         host_xm_defines="OMF=\\\"elf\\\" ${host_xm_defines}"
         extra_modes=pic30/pic30-modes.h
         LANGUAGES="c"
